src/components/Personalization/createOnClickHandler.js

```diff
@@ -48,7 +48,7 @@ export default ({
           event,
           decisionsMeta,
           PropositionEventType.INTERACT,
-          eventLabel
+          eventLabel ? { label: eventLabel } : undefined
```

Comment:
why do we need to pass `undefined` here?

Reply:
because there should only be an object passed in here if eventLabel is defined.

see https://github.com/adobe/alloy/blob/in-browser-messages/src/components/Personalization/event.js#L33